src/promptflow/promptflow/_sdk/_constants.py

```diff
@@ -77,6 +77,7 @@ def _prepare_home_dir() -> Path:
 PF_SERVICE_LOG_FILE = "pfs.log"
 PF_SERVICE_HOUR_TIMEOUT = 1
 PF_SERVICE_MONITOR_SECOND = 60
+PF_SERVICE_WAITRESS_THREADS = 16
```

Comment:
maybe a field in pf config ?

Reply:
config can be another PR? this just work like a default value